workflow-templates/im-test-cypress-schedule.yml

```diff
@@ -0,0 +1,130 @@
+# Workflow Code: SurprisedHedgehog_v22    DO NOT REMOVE
+# Purpose:
+#    Runs Cypress UI Health Check tests in Github Actions on a schedule.
+#    With the workflow you specify when to run this test.
+#
+# Frequency:
+#    - This workflow is intended to be reusable, and used once per repository.
+#
+# Projects to use this Template with:
+#    - Testing projects that use Cypress.
+#
+# TODO Prerequisites:
+#    - Add secret APPLICATIONINSIGHTS_CONNECTION_STRING to every environment you are testing.
+#    - Update your cypress test config to include the im.exporttoappinsights package. Instructions found here: https://github.com/im-practices/techhub/blob/main/docs/Azure/content/Health-Checks/Cypress-UI-Health-Check.md
+#    - If the project contains a local nuget.config remove it.  It interferes with restoring packages on the GitHub Action runners.
+#    - In order to use this workflow a few changes need to be made to the cypress test configuration:
+#      1. Make the following npm package changes in your cypress test directory
+#         npm uninstall mocha-teamcity-reporter
+#         npm install --save-dev mochawesome mochawesome-merge mochawesome-report-generator
+#      2. Modify the cypress run section of the ci script by replacing the '--reporter ./mocha-teamcity-reporter' with '--reporter mochawesome --reporter-options overwrite=false,html=false,json=true'. For instance:
+#         old: "cy:ci" : "....&& cypress run --project ./ --reporter ./mocha-teamcity-reporter \"",
+#         new: "cy:ci" : "....&& cypress run --project ./ --reporter mochawesome --reporter-options overwrite=false,html=false,json=true \"",
+#      3. Add the following pre-ci script.  Your name will vary based on your cypress ci script name.  This removes results between test runs.
+#         "precy:ci": "rm -rf mochawesome-report/",
+#      4. Add the following script that merges all test json files and generates an html report.  Your name may vary based on your project conventions.  If the output file or results dir are changed, update the env values in the workflow.
```

Comment:
This step should be a step in this workflow.

Reply:
These steps should be part of the scripts in package.json file of the project and this script is being called in the workflow.
- name: Merge json files and create html report
        run: npm run ${{ env.CYPRESS_REPORT_SCRIPT }}

Reply:
Ok thanks.